### ISSUE TITLE
This is very helpful, thank you! Blastula via a Shiny app on shinyapps.io stopped working on May the 9th and after upgrading security settings it is back on working

### ISSUE THREAD

**Issue by mzuer** [rstudio/blastula#299]
This is very helpful, thank you! Blastula via a Shiny app on shinyapps.io stopped working on May the 9th and after upgrading security settings it is back on working

_Originally posted by @studiorevenue in https://github.com/rstudio/blastula/issues/282#issuecomment-1153736159_

**Comment by sanjmeh** [comment]
Yes I am working on a shiny app that has to periodically send out emails based on some alert conditions. So sending test emails has not been a problem for me. I am struggling in embedding tables in the body of the email, right now. But we can collaborate and help one another @mzuer .
